api/StaticRange.json

```
@@ -72,22 +72,22 @@
               "version_added": false
             },
             "opera": {
-              "version_added": "47"
+              "version_added": false
             },
             "opera_android": {
-              "version_added": "44"
+              "version_added": false
             },
             "safari": {
-              "version_added": false
+              "version_added": true
```

Looks like Safari actually doesn't support this, based upon running http://w3c-test.org/dom/ranges/StaticRange-constructor.html in the latest release (13.0.3).  Let's set this to `false` as well?

```suggestion
              "version_added": false
```

---

> Looks like Safari actually doesn't support this, based upon running http://w3c-test.org/dom/ranges/StaticRange-constructor.html in the latest release (13.0.3). Let's set this to `false` as well?

It’s supported in current Safari Technology Preview. That means it’ll ship in Safari 13.1. So I’ve gone ahead and set the value to `"13.1"`

---

"13.1" is not valid for Safari iOS, however.  Can we set Safari iOS to `false`?

---

> "13.1" is not valid for Safari iOS, however. Can we set Safari iOS to `false`?

oops, yeah, thanks for catching that — changed now

---

Ah, apparently we don't have Safari Desktop "13.1" in our data either (I could have sworn we had added it)...

---

> Ah, apparently we don't have Safari Desktop "13.1" in our data either (I could have sworn we had added it)...

Oh… Should I create a PR for that?

---

After further review, it appears that it was decided not to include Safari 13.1 in general (see #4933).